### PR TITLE
Add session modification methods to PFCPClient API

### PR DESCRIPTION
**Pull request by EmanueleGallone**
~~Blocked by #2~~

### CODE REVIEW

File: pkg/pfcpsim/session/pfcp_session.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package session
```
Comment (osinstom) [inline pkg/pfcpsim/session/pfcp_session.go:1]: We should keep this in `pfcpsim` package - there is no need to create separate packages.

File: pkg/pfcpsim/pfcpsim.go [hunk 2]
```diff
@@ -209,6 +210,22 @@ func (c *PFCPClient) SendSessionEstablishmentRequest(pdrs []*ieLib.IE, fars []*i
 	return c.sendMsg(estReq)
 }
 
+func (c *PFCPClient) SendSessionModificationRequest(PeerSEID uint64, pdrs []*ieLib.IE, qers []*ieLib.IE ,fars []*ieLib.IE) error {
```
Comment (osinstom) [inline pkg/pfcpsim/pfcpsim.go:213]: As agreed offline, the PR should only contain `SendSessionModificationRequest`. Please split this PR.

**Review by osinstom — CHANGES_REQUESTED** [review]

File: pkg/pfcpsim/pfcpsim.go (hunk 2 above)
Comment (EmanueleGallone) [reply pkg/pfcpsim/pfcpsim.go:212]: Done

File: pkg/pfcpsim/pfcpsim.go [hunk 3]
```diff
@@ -209,6 +209,22 @@ func (c *PFCPClient) SendSessionEstablishmentRequest(pdrs []*ieLib.IE, fars []*i
...
+	modifyReq := message.NewSessionModificationRequest(
+		0,
+		0,
+		PeerSEID,
+		c.getNextSequenceNumber(),
+		0,
+	)
+
+	modifyReq.CreatePDR = append(modifyReq.CreatePDR, pdrs...)
```
Comment (osinstom) [inline pkg/pfcpsim/pfcpsim.go:221]: Shouldn't this be `modifyReq.UpdatePDR` ?

Comment (EmanueleGallone) [reply pkg/pfcpsim/pfcpsim.go]: Right. while rolling back I forgot to update these. Thanks for catching this

**Review by osinstom — APPROVED** [review]

**Pull request merged**